docs/hosting/scaling/execution-data.md

```diff
@@ -58,13 +55,11 @@ n8n:
       - EXECUTIONS_DATA_SAVE_MANUAL_EXECUTIONS=false
 ```
 
-
 ## Enable data pruning
 
-You can enable data pruning to automatically delete executions after a given time period. If you don't set `EXECUTIONS_DATA_MAX_AGE`, 336 hours (14 days) is the default.
-
-You can choose to prune executions data before the time set in `EXECUTIONS_DATA_MAX_AGE`, using `EXECUTIONS_DATA_PRUNE_MAX_COUNT`. This sets a maximum number of executions to store in the database. Once you reach the limit, n8n starts to delete the oldest execution records. This can help with database performance issues, especially if you use SQLite.
+You can enable data pruning to automatically delete finished executions after a given time period. If you don't set `EXECUTIONS_DATA_MAX_AGE`, 336 hours (14 days) is the default.
 
+You can choose to prune finished executions data before the time set in `EXECUTIONS_DATA_MAX_AGE`, using `EXECUTIONS_DATA_PRUNE_MAX_COUNT`. This sets a maximum number of executions to store in the database. Once you reach the limit, n8n starts to delete the oldest execution records. This can help with database performance issues, especially if you use SQLite. This number is however not exact, and in some cases there might be more executions left in the database.
```

Comment:
"This number is however not exact, and in some cases there might be more executions left in the database." - this is vague. Do you mean n8n doesn't always stick to the maximum set? (I'm guessing if pruning only runs every so often, it could lead to the DB growing beyond the max temporarily? At any rate, can we give a reason?)